README.rst

```diff
@@ -0,0 +1,37 @@
+qp-fastp-minimap2 Qiita Plugin
+==============================
+
+|Build Status| |Coverage Status|
+
+`Qiita <https://github.com/biocore/qiita/>`__ (canonically pronounced *cheetah*)
+is an analysis environment for microbiome (and other "comparative -omics")
+datasets.
+
+This package includes the qp-fastp-minimap2 Qiita plugin.
+
+#TODO: Add a description of the processing commands added in this plugin.
```

Comment:
Should this be addressed?

Reply:
I'll remove, left by mistake, this is part of the cookiecutter template ...